sns/signals.py

```diff
@@ -0,0 +1,46 @@
+import logging
+from django.dispatch import Signal
+from django.dispatch import receiver
+from django.conf import settings
+from django.db.models.signals import post_save
+
+from sns.models import Bounce, Complaint
+
+from manager.models import Recipient
+
+logger = logging.getLogger(__name__)
+
+notification = Signal(providing_args=['notification', 'request'])
+
+feedback = Signal(providing_args=['instance', 'message'])
+
+@receiver(post_save, sender=Bounce)
```

Comment:
Consider adding comments above both of these receivers that give a brief explanation of what they do + why